akka-http/src/main/scala/akka/http/javadsl/server/RequestContext.scala

```diff
@@ -45,10 +53,19 @@ class RequestContext private (val delegate: scaladsl.server.RequestContext) {
       .fast.map(r ⇒ r: RouteResult)(akka.dispatch.ExecutionContexts.sameThreadExecutionContext).toJava
   }
 
+  def completeWith(response: HttpResponse): CompletionStage[RouteResult] = {
```

Comment:
This was missing from JavaDSL indeed.
Now that I look at it, it could be just `complete`?
`completeWith` we use in JavaDSL for Future stufff